src/components/LandingPage/FAQ/FAQ.tsx

```diff
@@ -24,19 +25,28 @@ export const FAQDropDown: React.FC<FAQDropDownProps> = (props) => {
   }
   return (
     <a
-      className="collapse border border-purple_hover rounded-box"
+      className="border collapse border-purple_hover rounded-box"
       onClick={toggleOpen}>
       <input type="checkbox" className={mounted ? 'hidden' : ''} />
-      <div className="collapse-title w-full bg-purple_300 hover:ease-in-out hover:duration-200 hover:bg-purple_hover hover:text-white text-purple_main border-none rounded-box rounded-b-none">
-        <p className="text-start leading-9 normal-case mr-auto text-md md:text-lg m-1">
+      <div
+        className={`flex justify-between items-center w-full text-white bg-transparent border-2 border-pink_200 collapse-title hover:ease-in-out hover:duration-200 hover:bg-purple_hover hover:text-white rounded-box transition-all duration-500 ease-in-out ${
+          opened ? 'rounded-b-none' : ''
```

Comment:
I don't think this condition affect the behavior of the dropdown can just keep it like:
```html
 <div className="flex justify-between items-center w-full text-white bg-transparent border-2 collapse-title hover:ease-in-out hover:duration-200 hover:bg-purple_hover hover:text-white rounded-box transition-all duration-500 ease-in-out rounded-b-none border-none">
```